molSimplify/Scripts/structgen.py

```diff
@@ -817,7 +817,7 @@ def openbabel_ffopt(ff, mol, connected, constopt, frozenats, frozenangles,
 
 
 def xtb_opt(ff, mol, connected, constopt, frozenats, frozenangles,
-            mlbonds, nsteps, spin=1, debug=False):
+            mlbonds, nsteps, spin=1, inertial=False, debug=False):
```

Comment:
This addition might seem unrelated. It came up due to the introduction of better test coverage.